### ISSUE TITLE
Courses section

### ISSUE THREAD

**Issue by mikebutts** [PascalB90/colmar-cademy#1]
https://github.com/PascalB90/colmar-cademy/blob/b69d37bf15e0c779d4d33f5801e39bd4a8ead951/index.html#L103

While this section looks very good, it only shows 4 courses @ 100%, if I zoom out to 90% it shows all 6, so I am not sure how you set this. For the record I am using a 15.4" 4k laptop so it should show the 6 no problem.

**Comment by PascalB90** [comment]
yes , you are right the website was developed on a 4k screen , i will change it that show 6 on a normal desktop :-) thanks a lot
